client/src/Components/Login.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+
+axios.defaults.baseURL = 'http://localhost:5000'
+axios.defaults.withCredentials = true
+
+export default function Login() {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const [error, setError] = useState(null);
+  const [login, setLogin] = useState(null)
+
+  const handleLogin = () => {
+    axios.post(`${axios.defaults.baseURL}/login`, {
```

Comment:
Nice that you tried out Axios!
Actually, you do not need to set the baseURL anymore, because you already did that in line 4!
So with Axios you can set the baseURL once for ALL your requests. That is super nice.
So in this case you can just do `axios.post("/login")`